infrastructure/migrations/0019_project_latest_implementation_year.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-09-23 14:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import infrastructure.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('infrastructure', '0018_auto_20210915_1645'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='latest_implementation_year',
+            field=models.ForeignKey(default=infrastructure.models.get_finanial_year_default, on_delete=django.db.models.deletion.CASCADE, to='infrastructure.FinancialYear'),
+        ),
+    ]
```

Comment:
A better approach than providing a default method:

- add the field with null=True
- run some python code which, if projects exist, fetches the year 2019/20 (assume it exists), and update the projects with that year selected
- change the field to null=False

Reply:
Reworked the migrations further and included this change